PSScriptAnalyzerRules.psd1

```diff
@@ -0,0 +1,84 @@
+@{
+    Severity     = @('Error')
+    IncludeRules = @(
+        'PSUseCompatibleSyntax',
+        'PSAvoidUsingCmdletAliases',
+        'PSAvoidUsingWriteHost',
+        'PSAvoidDefaultValueSwitchParameter',
+        'PSReservedCmdletChar',
+        'PSReservedParams',
+        'PSAvoidUsingUserNameAndPassWordParams',
+        'PSAvoidUsingPlaintTextForPassword',
+        'PSAvoidUsingWMICmdlet',
+        'PSAvoidUsingWriteHost',
+        'PSMisleadingBacktick',
+        'PSMissingModuleManifestField',
+        'PSPossibleIncorrectComparisonWithNull',
+        'PSUseApprovedVerbs',
+        'PSUseOutputTypeCorrectly',
+        'PSShouldProcess',
+        'PSUserToExportFieldsInManifest',
+        'PSUseSingularNouns',
+        'PSAvoidUsingInvokeExpression',
+        'PSUseShouldProcessForStateChangingFunctions',
```

Comment:
```suggestion
```
I don't prefer this, as a New or Remove operation on the OpenAI API does not imply a destructive change to the system. This rule, which is based solely on the verb of the command, has too many false positives.